src/modules/cart/controller/cartControllers.ts

```diff
@@ -180,8 +200,23 @@ const buyerCreateUpdateCart = async (req: ExtendRequest, res: Response) => {
   }
 };
 
+const buyerClearcart = async (req: ExtendRequest, res: Response) => {
+  try {
+    console.log(req, res);
+    const cart = await cartRepositories.getCartByUserId(req.user.id);
```

Comment:
This approach will clear random buyer's cart since buyer have many carts.
Why one endpoint?

user have many carts, better make 2 endpoints, 
1. Clear cart by providing cartId in params
2. Clear all buyer's carts